docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -279,6 +297,23 @@ export default class MethodPage extends React.PureComponent<Props, State> {
     });
   };
 
+  private onCopy = () => {
+    const response = this.state.debugResponse;
+    const textArea = document.createElement('textarea');
```

Comment:
Does it work with `textarea.style.display = 'none'` or `opacity = 0`? I wonder if there's a chance for it to flash.

Reply:
I think it shouldn't flash but to be safe maybe better to add `textArea.style.opacity = '0.0';`